src/app/optimizer/optimizer.component.ts

```diff
@@ -15,12 +15,14 @@ import {ResultsTableComponent} from './results-table/results-table.component';
   providers: [OptimizerDataService, HTTP_PROVIDERS]
 })
 export class OptimizerComponent {
-  optimalAllocs: Object;
+  optimalAllocs;
```

Comment:
Can we declare the type here? We think that declaring an array would be appropriate. Can we do Array<>